django_comments_xtd/__init__.py

```diff
@@ -23,7 +23,7 @@ def get_form():
 
 def get_version():
     version = '%s.%s' % (VERSION[0], VERSION[1])
```

Comment:
Is there a reason we only handle major and minor here? Could we simply grab the patch version all the time like so?
`version = '%s.%s.%s' % (VERSION[0], VERSION[1], VERSION[2])

Reply:
Yes, that's the part to remove, line 25, and line 26.
We want the function to retrieve at least `<major>.<minor>.<patch>`.

Reply:
Addressed